libafl/src/observers/map.rs

```diff
@@ -558,7 +752,7 @@ where
     }
 }
 
-impl<'a, T> StdMapObserver<'a, T, false>
+impl<'a, T> StdMapObserver<'a, T, false, false, false>
```

Comment:
Can't say this is getting easier for newcomers :D :D 

But I guess it's a price we'll have to pay to make it easier on the other end

Reply:
Okay, I made a wrapper type that should Just Work™ with all MapObservers and changed how MapFeedback works. We'll need to change this in a couple of other places, but this should make it basically zero implementation effort for people making map observers.